demo/standalone/util.rb

```diff
@@ -99,17 +99,19 @@ def call(env)
 end
 
 # Add persistent storage volumes
-def attach_volumes(node, num_volumes, volume_size)
+def attach_volumes(node, disk_sizes)
 
     node.vm.provider :virtualbox do |v, override|
```

Comment:
Yes i use VMware and others here use parallels.. I was just wondering if you missed updating those. LGTM